### PR TITLE
Documentation for new tracking

### PR DESCRIPTION
**Pull request by davidsevcik**
Documentation reflects the changes in https://github.com/reevoo/mark/pull/601 and https://github.com/reevoo/mark/pull/602

The old way of tracking still works and is tested as part of https://github.com/reevoo/mark/pull/602 as well as the updated reevoomark-loaded JS code.

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
*If you're new, read the [guidelines]
(https://github.com/reevoo/guidelines/blob/master/pull_requests.md).*

Good code is **tested**, **designed for change**, **easy to understand**
and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

File: docs/reevoomark/getting-started.markdown [hunk 1]
```diff
@@ -27,13 +27,15 @@ For this example you need to replace ```TRKREF``` with the account code provided
...
 
 <script id="reevoomark-loader" type="text/javascript" charset="utf-8">
-  (function() {
-    var script = document.createElement('script');
-    script.type = 'text/javascript';
-    script.src = '//cdn.mark.reevoo.com/assets/reevoo_mark.js';
-    var s = document.getElementById('reevoomark-loader');
-    s.parentNode.insertBefore(script, s);
-  })();
+  (function(w,d,u,i,f,s,l) {
```
Comment (robinbortlik) [inline docs/reevoomark/getting-started.markdown]: You do not pass ```s``` and ```l``` to the function. So why to declare them?

**Comment by robinbortlik** [comment]
Apart of the weird ```function params``` :) it looks good to me.

File: docs/reevoomark/getting-started.markdown (hunk 1 above)
Comment (errm) [inline docs/reevoomark/getting-started.markdown]: could we also think of better argument names . . . .

Comment (davidsevcik) [inline docs/reevoomark/getting-started.markdown]: I've copied the style from Google Analytic in the attempt to make the code as short as possible..
```
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```
I was event thinking about merging some lines together. But we can vote about that :)

Comment (errm) [inline docs/reevoomark/getting-started.markdown]: this seems fairly pointless.  It makes it impossible to read, if someone really cares this much they could do it themselves.

**Pull request merged**